### PR TITLE
Added ci test with all switches off

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [X] Added ci test with all switches off to avoid bugs when making changes to the pipeline
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,7 +34,8 @@ Release of genomic-medicine-sweden/tomte, created with the [nf-core](https://nf-
 - Shortened name of DROP output files [#79](https://github.com/genomic-medicine-sweden/tomte/pull/79)
 - Merging of vcfs has been moved to after bootstrapAnn [#81](https://github.com/genomic-medicine-sweden/tomte/pull/81)
 - Substituted bgzip and tabix modules by bgzip_tabix module [#85](https://github.com/genomic-medicine-sweden/tomte/pull/85)
-- Updated module input channels in the GATK variant calling subworkflow [#89](https://github.com/genomic-medicine-sweden/tomte/pull/85)
+- Updated module input channels in the GATK variant calling subworkflow [#89](https://github.com/genomic-medicine-sweden/tomte/pull/89)
+- Subsample and downsample switches [#97](https://github.com/genomic-medicine-sweden/tomte/pull/97)
```
Comment (jemten) [inline CHANGELOG.md:38]: Could you make a new heading in the CHANGELOG and add these fixes to that one as these fixes aren't part of the Rudolph release

**Review by jemten — COMMENTED** [review]
Looks good. Just need to move the changes in the CHANGELOG

File: CHANGELOG.md (hunk 1 above)
Comment (Lucpen) [reply CHANGELOG.md:38]: That's true!

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**